app/controllers/fridges_controller.rb

```diff
@@ -1,6 +1,31 @@
 class FridgesController < ApplicationController
 
-  def show
-  end
+#   def index
+#     @fridges = Fridge.where(user: current_user)
+#   end
+  
+
+#   def new 
+#     @fridge = Fridge.new
+#   end
```

Comment:
i think here is some authorize missing - only a signed in user can add a fridge. but @D-Zaleski knows more if this is actually the case here as well.

Reply:
this file is commented out, still in progress :) 
Just to be clear a few files went true with branch because i was working on them before. After meet with Vlad we work in set the fridge as default.
Because of that,  we are having some code being pushed still in progress ;)